### PR TITLE
cmd/gitbase: make squash rule active by default

### PR DESCRIPTION
**Pull request by erizocosmico**
Depends on #338

### CODE REVIEW

**Comment by ajnavarro** [comment]
Can you rebase and update documentation please?

**Comment by erizocosmico** [comment]
Yup

**Comment by erizocosmico** [comment]
Done

**Comment by ajnavarro** [comment]
We should change https://github.com/src-d/gitbase/blob/master/docs/using-gitbase/configuration.md#command-line-arguments too

File: cmd/gitbase/main.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package main
 import (
 	"os"
 
+	"github.com/sirupsen/logrus"
```
Comment (smola) [inline cmd/gitbase/main.go]: Move this one to external dependencies block.

File: docs/using-gitbase/configuration.md [hunk 2]
```diff
@@ -55,4 +54,5 @@ Help Options:
       -P, --password= Password used for connection
           --pilosa=   URL to your pilosa server (default: http://localhost:10101)
       -i, --index=    Directory where the gitbase indexes information will be persisted. (default: /var/lib/gitbase/index)
+          --nosquash  Disables the table squashing.
```
Comment (smola) [inline docs/using-gitbase/configuration.md]: Maybe `--no-squash`?

**Comment by erizocosmico** [comment]
Done @smola

**Comment by ajnavarro** [comment]
@erizocosmico I meant this too: https://github.com/src-d/gitbase/pull/345/files#diff-72163f02ff6598975a3c6cc1e5abab86L39

**Comment by erizocosmico** [comment]
Done

**Comment by ajnavarro** [comment]
@erizocosmico can you rebase please?

**Comment by erizocosmico** [comment]
Yup

**Comment by erizocosmico** [comment]
Done

**Pull request merged**